### DISCUSSION TITLE
Automatic Equation Numbering

### DISCUSSION THREAD

**Discussion by GoBigorGoHome** [marp-team/marp#452] [Ideas]
Please consider configuring mathjax in marp-for-vscode (or allow users to do that via options) so that equations are auto-numbered.

See https://docs.mathjax.org/en/latest/input/tex/eqnumbers.html for details.

Currently, when using mathjax, to number an equation, one has to write \tag{1} manually.

**Comment by czy140232** [comment]
TRUE. The \ref{equation number} seems to fail in marp-for-vscode as well, which makes math typing weirdly difficult.
